packages/protocol/installed_contracts/bytes/contracts/AssertBytes.sol

```diff
@@ -0,0 +1,200 @@
+/*
```

Comment:
In the past we've added external contracts as npm packages if possible. Is there a reason why we need to import it this way, adding another tool to our dependency management?

Reply:
Probably not. I'll try doing it through npm.